WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -210,6 +202,24 @@ class NoticePresenter: NSObject {
     }
 }
 
+private extension UIWindow {
+    /// Returns a rectangle based on this window offset such that a new window created
+    /// with this frame will not overtake the status bar responsibilities
+    ///
+    /// - Returns: CGRect based on this window's frame
+    /// - Note: Turns out that a small alteration to the frame is enough to accomplish this.
+    func offsetToAvoidStatusBar() -> CGRect {
+        return self.frame.inset(by: UIEdgeInsets(top: Offsets.minimalEdgeOffset,
```

Comment:
_Pedantic:_ Because the inset values are all the same, [`insetBy(dx:dy:)`](https://developer.apple.com/documentation/coregraphics/cgrect/1454218-insetby) is arguable more concise.

Reply:
Ah, nice catch. I was using the more verbose version to test out possibilities. That will be a nice cleanup 👍